### PR TITLE
Fix: rotateAuthAccount failed when called twice

### PR DESCRIPTION
**Pull request by btspoony**
When `giveOwnership` and `addOwnedAccount` are executed in the same transaction
`rotateAuthAccount` will be called twice.
However, during the second call, since authAcctPath is the same, it will cause `acct.linkAccount` to return `nil` as a result.

Personal opinion, it would be better to change `HybridCustody` overall to CapCons coding style.

### CODE REVIEW

File: contracts/HybridCustody.cdc [hunk 1]
```diff
@@ -1023,7 +1023,10 @@ pub contract HybridCustody {
             // NOTE: This path cannot be sufficiently randomly generated, an app calling this function could build a
             // capability to this path before it is made, thus maintaining ownership despite making it look like they
             // gave it away. Until capability controllers, this method should not be fully trusted.
-            let authAcctPath = "HybridCustodyRelinquished".concat(HybridCustody.account.address.toString()).concat(getCurrentBlock().height.toString())
+            let authAcctPath = "HybridCustodyRelinquished"
+                .concat(HybridCustody.account.address.toString())
+                .concat(getCurrentBlock().height.toString())
+                .concat((pathsToUnlink.length + 1).toString()) // ensure that the path is different from the previous one
```
Comment (austinkline) [inline contracts/HybridCustody.cdc:1029]: Are we able to use [unsafeRandom](https://developers.flow.com/cadence/language/built-in-functions#unsaferandom) here? Or does it make the same number when called twice?

Comment (btspoony) [reply contracts/HybridCustody.cdc:1029]: I initially made modifications using unsafeRandom, but it threw an error in the simulator. It seems that it has just been fixed and hasn't been merged into the release yet.

Personally, I also prefer to use `unsafeRandom`.

Comment (btspoony) [reply contracts/HybridCustody.cdc:1029]: FYI,  `unsafeRandom` will cause a Go exception ([Issue](https://github.com/onflow/flow-cli/issues/1174)) that is supposed to be fixed by the [PR](https://github.com/onflow/flow-emulator/pull/459).

I will update to use `unsafeRandom` after the fix released

``` shell
11:12PM WRN ERR [4dc519] [Error Code: 1101] error caused by: 1 error occurred:
        * transaction execute failed: [Error Code: 1101] cadence runtime error: Execution failed:
error: internal error: runtime error: invalid memory address or nil pointer dereference
goroutine 1129 [running]:
runtime/debug.Stack()
        /usr/local/go/src/runtime/debug/stack.go:24 +0x64
github.com/onflow/cadence/runtime/errors.NewUnexpectedErrorFromCause(...)
        /go/pkg/mod/github.com/onflow/cadence@v0.40.0/runtime/errors/errors.go:152
github.com/onflow/cadence/runtime/interpreter.asCadenceError({0x104a54540, 0x105d8d0f0})
        /go/pkg/mod/github.com/onflow/cadence@v0.40.0/runtime/interpreter/interpreter.go:570 +0x1b4
github.com/onflow/cadence/runtime/interpreter.(*Interpreter).RecoverErrors(0x14001417140, 0x104d49870)
        /go/pkg/mod/github.com/onflow/cadence@v0.40.0/runtime/interpreter/interpreter.go:522 +0x40
panic({0x104a54540, 0x105d8d0f0})
        /usr/local/go/src/runtime/panic.go:884 +0x204
github.com/onflow/cadence/runtime/errors.WrapPanic.func1()
        /go/pkg/mod/github.com/onflow/cadence@v0.40.0/runtime/errors/wrappanic.go:31 +0x90
panic({0x104a54540, 0x105d8d0f0})
        /usr/local/go/src/runtime/panic.go:884 +0x204
github.com/onflow/flow-go/fvm/environment.(*randomGenerator).createPRG(0x14000f99ea0)
        /go/pkg/mod/github.com/onflow/flow-go@v0.31.1-0.20230808172820-f074502a67e3/fvm/environment/random_generator.go:86 +0x24
github.com/onflow/flow-go/fvm/environment.(*randomGenerator).UnsafeRandom(0x14000f99ea0)
        /go/pkg/mod/github.com/onflow/flow-go@v0.31.1-0.20230808172820-f074502a67e3/fvm/environment/random_generator.go:118 +0x100
github.com/onflow/flow-go/fvm/environment.parseRestrict1Ret[...]({0x12db27528?, 0x140010e9530?}, {0x1042dafba?, 0x14001397338?}, 0x14001397310?)
        /go/pkg/mod/github.com/onflow/flow-go@v0.31.1-0.20230808172820-f074502a67e3/fvm/environment/parse_restricted_checker.go:101 +0x64
github.com/onflow/flow-go/fvm/environment.ParseRestrictedRandomGenerator.UnsafeRandom(...)
        /go/pkg/mod/github.com/onflow/flow-go@v0.31.1-0.20230808172820-f074502a67e3/fvm/environment/random_generator.go:62
github.com/onflow/cadence/runtime.(*interpreterEnvironment).UnsafeRandom(0x14001301500?)
        /go/pkg/mod/github.com/onflow/cadence@v0.40.0/runtime/environment.go:223 +0x28
github.com/onflow/cadence/runtime/stdlib.NewUnsafeRandomFunction.func1.1.1()
        /go/pkg/mod/github.com/onflow/cadence@v0.40.0/runtime/stdlib/random.go:59 +0x38
github.com/onflow/cadence/runtime/errors.WrapPanic(0x140013973f8?)
        /go/pkg/mod/github.com/onflow/cadence@v0.40.0/runtime/errors/wrappanic.go:43 +0x40
github.com/onflow/cadence/runtime/stdlib.NewUnsafeRandomFunction.func1.1()
        /go/pkg/mod/github.com/onflow/cadence@v0.40.0/runtime/stdlib/random.go:58 +0x64
github.com/onflow/cadence/runtime/interpreter.NewUInt64Value({0x104d5cca0?, 0x14001417140?}, 0x140013974d0?)
        /go/pkg/mod/github.com/onflow/cadence@v0.40.0/runtime/interpreter/value.go:10075 +0x60
github.com/onflow/cadence/runtime/stdlib.NewUnsafeRandomFunction.func1({{{0x104d7a100, 0x14001766a80}, {0x104d68968, 0x1400102f260}}, 0x0, 0x0, 0x140013607d0, 0x14001417140, {0x0, 0x0, ...}, ...})
        /go/pkg/mod/github.com/onflow/cadence@v0.40.0/runtime/stdlib/random.go:53 +0x88
github.com/onflow/cadence/runtime/interpreter.(*HostFunctionValue).invoke(0x14001417140?, {{{0x104d7a100, 0x14001766a80}, {0x104d68968, 0x1400102f260}}, 0x0, 0x0, 0x140013607d0, 0x14001417140, {0x0, ...}, ...})
        /go/pkg/mod/github.com/onflow/cadence@v0.40.0/runtime/interpreter/value_function.go:256 +0x94
github.com/onflow/cadence/runtime/interpreter.(*Interpreter).invokeFunctionValue(0x1400102f260?, {0x104d8dea0, 0x140006241e0}, {0x0?, 0x18000000007798?, 0x12deb1c30?}, {0x0?, 0x103b73dd8?, 0x1400004e900?}, {0x0?, ...}, ...)
        /go/pkg/mod/github.com/onflow/cadence@v0.40.0/runtime/interpreter/interpreter_invocation.go:124 +0x170
github.com/onflow/cadence/runtime/interpreter.(*Interpreter).visitInvocationExpressionWithImplicitArgument(0x14001417140, 0x1400102f260, 0x0)
        /go/pkg/mod/github.com/onflow/cadence@v0.40.0/runtime/interpreter/interpreter_expression.go:975 +0x6b4
github.com/onflow/cadence/runtime/interpreter.(*Interpreter).VisitInvocationExpression(0x104a39d60?, 0x140013ba7b0?)
        /go/pkg/mod/github.com/onflow/cadence@v0.40.0/runtime/interpreter/interpreter_expression.go:880 +0x20
github.com/onflow/cadence/runtime/ast.AcceptExpression[...]({0x104d850b8?, 0x1400102f260}, {0x104d94370, 0x14001417140})
        /go/pkg/mod/github.com/onflow/cadence@v0.40.0/runtime/ast/visitor.go:230 +0x21c
github.com/onflow/cadence/runtime/interpreter.(*Interpreter).evalExpression(...)
        /go/pkg/mod/github.com/onflow/cadence@v0.40.0/runtime/interpreter/interpreter_expression.go:279
github.com/onflow/cadence/runtime/interpreter.(*Interpreter).assignmentGetterSetter.func1(0x40?)
        /go/pkg/mod/github.com/onflow/cadence@v0.40.0/runtime/interpreter/interpreter_expression.go:53 +0x38
github.com/onflow/cadence/runtime/interpreter.(*Interpreter).visitVariableDeclaration(0x14001417140, 0x14000faaf00, 0x14001397d08)
        /go/pkg/mod/github.com/onflow/cadence@v0.40.0/runtime/interpreter/interpreter_statement.go:502 +0xec
github.com/onflow/cadence/runtime/interpreter.(*Interpreter).VisitVariableDeclaration(0x104d89cd8?, 0x1400102ba40?)
        /go/pkg/mod/github.com/onflow/cadence@v0.40.0/runtime/interpreter/interpreter_statement.go:462 +0x34
github.com/onflow/cadence/runtime/ast.AcceptStatement[...]({0x104d80ac0?, 0x14000faaf00}, {0x104d8f3c0, 0x14001417140})
        /go/pkg/mod/github.com/onflow/cadence@v0.40.0/runtime/ast/visitor.go:143 +0x354
github.com/onflow/cadence/runtime/interpreter.(*Interpreter).evalStatement(0x14001417140, {0x104d80ac0?, 0x14000faaf00})
        /go/pkg/mod/github.com/onflow/cadence@v0.40.0/runtime/interpreter/interpreter_statement.go:57 +0x130
github.com/onflow/cadence/runtime/interpreter.(*Interpreter).visitStatements(0x104d6ba40?, {0x1400151b500?, 0x3, 0x140013b2f70?})
        /go/pkg/mod/github.com/onflow/cadence@v0.40.0/runtime/interpreter/interpreter_statement.go:63 +0x68
github.com/onflow/cadence/runtime/interpreter.(*Interpreter).invokeInterpretedFunctionActivated.func2()
        /go/pkg/mod/github.com/onflow/cadence@v0.40.0/runtime/interpreter/interpreter_invocation.go:173 +0x30
github.com/onflow/cadence/runtime/interpreter.(*Interpreter).visitFunctionBody(0x14001417140, {0x0, 0x0, 0x0}, {0x0, 0x0, 0x140013b2ff8?}, 0x14001398030, {0x0, 0x0, ...}, ...)
        /go/pkg/mod/github.com/onflow/cadence@v0.40.0/runtime/interpreter/interpreter.go:782 +0x170
github.com/onflow/cadence/runtime/interpreter.(*Interpreter).invokeInterpretedFunctionActivated(0x14001417140?, 0x140013bfc00?, {0x14001361270?, 0x1?, 0x140014173e0?})
        /go/pkg/mod/github.com/onflow/cadence@v0.40.0/runtime/interpreter/interpreter_invocation.go:169 +0x11c
github.com/onflow/cadence/runtime/interpreter.(*Interpreter).invokeInterpretedFunction(0x14001417140, 0x140013bfc00, {{{0x0, 0x0}, {0x0, 0x0}}, 0x14001361290, 0x0, 0x0, 0x14001417140, ...})
        /go/pkg/mod/github.com/onflow/cadence@v0.40.0/runtime/interpreter/interpreter_invocation.go:148 +0x1dc
github.com/onflow/cadence/runtime/interpreter.(*InterpretedFunctionValue).invoke(...)
        /go/pkg/mod/github.com/onflow/cadence@v0.40.0/runtime/interpreter/value_function.go:126
github.com/onflow/cadence/runtime/interpreter.(*Interpreter).declareTransactionEntryPoint.func1({{{0x0, 0x0}, {0x0, 0x0}}, 0x14001361290, 0x0, 0x0, 0x14001417140, {0x14001361270, 0x1, ...}, ...})
        /go/pkg/mod/github.com/onflow/cadence@v0.40.0/runtime/interpreter/interpreter_transaction.go:107 +0x24c
github.com/onflow/cadence/runtime/interpreter.(*HostFunctionValue).invoke(0x14001417140?, {{{0x0, 0x0}, {0x0, 0x0}}, 0x0, 0x0, 0x0, 0x14001417140, {0x14001361270, ...}, ...})
        /go/pkg/mod/github.com/onflow/cadence@v0.40.0/runtime/interpreter/value_function.go:256 +0x94
github.com/onflow/cadence/runtime/interpreter.(*Interpreter).InvokeExternally(0x104c17c80?, {0x104d8dea0, 0x14001382c60}, 0x1400131cfc0?, {0x14001361260?, 0x1?, 0x14001398688?})
        /go/pkg/mod/github.com/onflow/cadence@v0.40.0/runtime/interpreter/interpreter.go:473 +0x270
github.com/onflow/cadence/runtime/interpreter.(*Interpreter).InvokeTransaction(0x14001417140, 0x14001361260?, {0x14001361260, 0x1, 0x1})
        /go/pkg/mod/github.com/onflow/cadence@v0.40.0/runtime/interpreter/interpreter.go:515 +0x200
github.com/onflow/cadence/runtime.(*interpreterTransactionExecutor).transactionExecutionFunction.func1(0x14001417140)
        /go/pkg/mod/github.com/onflow/cadence@v0.40.0/runtime/transaction_executor.go:268 +0x2ac
github.com/onflow/cadence/runtime.(*interpreterEnvironment).Interpret.func1()
        /go/pkg/mod/github.com/onflow/cadence@v0.40.0/runtime/environment.go:1021 +0xa8
github.com/onflow/cadence/runtime.reportMetric(0x140013989a0, {0x12db2a6d8?, 0x140000cb1e0?}, 0x14001398988)
        /go/pkg/mod/github.com/onflow/cadence@v0.40.0/runtime/runtime.go:205 +0x5c
github.com/onflow/cadence/runtime.(*interpreterEnvironment).Interpret(0x14001154090, {0x104d7a100, 0x14001766a80}, 0x30?, 0x14001382300)
        /go/pkg/mod/github.com/onflow/cadence@v0.40.0/runtime/environment.go:1015 +0xb4
github.com/onflow/cadence/runtime.(*interpreterTransactionExecutor).execute(0x14000926360)
        /go/pkg/mod/github.com/onflow/cadence@v0.40.0/runtime/transaction_executor.go:223 +0x104
github.com/onflow/cadence/runtime.(*interpreterTransactionExecutor).Execute.func1()
        /go/pkg/mod/github.com/onflow/cadence@v0.40.0/runtime/transaction_executor.go:80 +0x24
sync.(*Once).doSlow(0x140012afac0?, 0x14001360bc0?)
        /usr/local/go/src/sync/once.go:74 +0x104
sync.(*Once).Do(...)
        /usr/local/go/src/sync/once.go:65
github.com/onflow/cadence/runtime.(*interpreterTransactionExecutor).Execute(0x14000926360?)
        /go/pkg/mod/github.com/onflow/cadence@v0.40.0/runtime/transaction_executor.go:79 +0x50
github.com/onflow/flow-go/fvm/runtime.WrappedCadenceExecutor.Execute({{0x104d6e170?, 0x14000926360?}})
        /go/pkg/mod/github.com/onflow/flow-go@v0.31.1-0.20230808172820-f074502a67e3/fvm/runtime/wrapped_cadence_runtime.go:90 +0x2c
github.com/onflow/flow-go/fvm.(*transactionExecutor).normalExecution(0x1400036af00)
        /go/pkg/mod/github.com/onflow/flow-go@v0.31.1-0.20230808172820-f074502a67e3/fvm/transactionInvoker.go:334 +0x14c
github.com/onflow/flow-go/fvm.(*transactionExecutor).ExecuteTransactionBody(0x1400036af00)
        /go/pkg/mod/github.com/onflow/flow-go@v0.31.1-0.20230808172820-f074502a67e3/fvm/transactionInvoker.go:231 +0x4c
github.com/onflow/flow-go/fvm.(*transactionExecutor).execute(0x1400036af00?)
        /go/pkg/mod/github.com/onflow/flow-go@v0.31.1-0.20230808172820-f074502a67e3/fvm/transactionInvoker.go:223 +0x24
github.com/onflow/flow-go/fvm.(*transactionExecutor).Execute(0x6ce1e2af59833d48?)
        /go/pkg/mod/github.com/onflow/flow-go@v0.31.1-0.20230808172820-f074502a67e3/fvm/transactionInvoker.go:140 +0x20
github.com/onflow/flow-go/fvm.Run({0x104d75830?, 0x1400036af00})
        /go/pkg/mod/github.com/onflow/flow-go@v0.31.1-0.20230808172820-f074502a67e3/fvm/fvm.go:83 +0xac
github.com/onflow/flow-go/fvm.(*VirtualMachine).Run(_, {0x0, 0x186a0, 0xffffffffffffffff, 0x3e80, 0xf424000, 0x1312d00, {0x1, 0x1, 0x3e8, ...}, ...}, ...)
        /go/pkg/mod/github.com/onflow/flow-go@v0.31.1-0.20230808172820-f074502a67e3/fvm/fvm.go:182 +0x2b4
github.com/onflow/flow-emulator/emulator.(*pendingBlock).ExecuteNextTransaction(_, _, {0x0, 0x186a0, 0xffffffffffffffff, 0x3e80, 0xf424000, 0x1312d00, {0x1, 0x1, ...}, ...})
        /go/pkg/mod/github.com/onflow/flow-emulator@v0.54.0/emulator/pendingBlock.go:186 +0x23c
github.com/onflow/flow-emulator/emulator.(*Blockchain).executeNextTransaction(_, {0x0, 0x186a0, 0xffffffffffffffff, 0x3e80, 0xf424000, 0x1312d00, {0x1, 0x1, 0x3e8, ...}, ...})
        /go/pkg/mod/github.com/onflow/flow-emulator@v0.54.0/emulator/blockchain.go:1209 +0x354
github.com/onflow/flow-emulator/emulator.(*Blockchain).executeBlock(0x14000680900)
        /go/pkg/mod/github.com/onflow/flow-emulator@v0.54.0/emulator/blockchain.go:1165 +0x178
github.com/onflow/flow-emulator/emulator.(*Blockchain).executeAndCommitBlock(0x14000680900)
        /go/pkg/mod/github.com/onflow/flow-emulator@v0.54.0/emulator/blockchain.go:1310 +0x28
github.com/onflow/flow-emulator/emulator.(*Blockchain).SendTransaction(0x14000680900, 0x14000e50240)
        /go/pkg/mod/github.com/onflow/flow-emulator@v0.54.0/emulator/blockchain.go:1089 +0xc8
github.com/onflow/flow-emulator/adapters.(*AccessAdapter).SendTransaction(0x14001ba0648, {0x20a6cd3f8d2c945c?, 0x2d42fcadc172188?}, 0x14000e50240)
        /go/pkg/mod/github.com/onflow/flow-emulator@v0.54.0/adapters/access.go:444 +0x1c0
github.com/onflow/flow-go/engine/access/rest/routes.CreateTransaction(0x140005da200, {0x104d96088, 0x14001ba0648}, {0x104d80c50, 0x1400063a190})
        /go/pkg/mod/github.com/onflow/flow-go@v0.31.1-0.20230808172820-f074502a67e3/engine/access/rest/routes/transactions.go:59 +0x124
github.com/onflow/flow-go/engine/access/rest/routes.(*Handler).ServeHTTP(0x14000fd0780, {0x104d70150?, 0x140006b2080}, 0x14000802a00)
        /go/pkg/mod/github.com/onflow/flow-go@v0.31.1-0.20230808172820-f074502a67e3/engine/access/rest/routes/handler.go:77 +0x35c
github.com/slok/go-http-metrics/middleware/std.Handler.func1.1()
        /go/pkg/mod/github.com/slok/go-http-metrics@v0.10.0/middleware/std/std.go:27 +0x38
github.com/slok/go-http-metrics/middleware.Middleware.Measure({{{0x12dedb5e0, 0x14000d4cc90}, {0x0, 0x0}, 0x0, 0x0, 0x0}}, {0x0, 0x0}, {0x104d78b80, ...}, ...)
        /go/pkg/mod/github.com/slok/go-http-metrics@v0.10.0/middleware/middleware.go:117 +0x20c
github.com/slok/go-http-metrics/middleware/std.Handler.func1({0x104d6e680?, 0x1400025c618}, 0x14000802a00)
        /go/pkg/mod/github.com/slok/go-http-metrics@v0.10.0/middleware/std/std.go:26 +0x164
net/http.HandlerFunc.ServeHTTP(0x0?, {0x104d6e680?, 0x1400025c618?}, 0x14000d4cc90?)
        /usr/local/go/src/net/http/server.go:2109 +0x38
github.com/onflow/flow-go/engine/access/rest/middleware.MetricsMiddleware.func1.1({0x104d6e680?, 0x1400025c540}, 0x14000802a00)
        /go/pkg/mod/github.com/onflow/flow-go@v0.31.1-0.20230808172820-f074502a67e3/engine/access/rest/middleware/metrics.go:25 +0x128
net/http.HandlerFunc.ServeHTTP(0x14000a73701?, {0x104d6e680?, 0x1400025c540?}, 0x6?)
        /usr/local/go/src/net/http/server.go:2109 +0x38
github.com/onflow/flow-go/engine/access/rest/middleware.commonQueryParamMiddleware.func1.1({0x104d6e680, 0x1400025c540}, 0x14000802a00)
        /go/pkg/mod/github.com/onflow/flow-go@v0.31.1-0.20230808172820-f074502a67e3/engine/access/rest/middleware/common_query_params.go:28 +0xec
net/http.HandlerFunc.ServeHTTP(0x617935d8?, {0x104d6e680?, 0x1400025c540?}, 0x6?)
        /usr/local/go/src/net/http/server.go:2109 +0x38
github.com/onflow/flow-go/engine/access/rest/middleware.commonQueryParamMiddleware.func1.1({0x104d6e680, 0x1400025c540}, 0x14000802a00)
        /go/pkg/mod/github.com/onflow/flow-go@v0.31.1-0.20230808172820-f074502a67e3/engine/access/rest/middleware/common_query_params.go:28 +0xec
net/http.HandlerFunc.ServeHTTP(0x0?, {0x104d6e680?, 0x1400025c540?}, 0x1040cd3f0?)
        /usr/local/go/src/net/http/server.go:2109 +0x38
github.com/onflow/flow-go/engine/access/rest/middleware.LoggingMiddleware.func1.1({0x104d72ac0?, 0x1400173a0e0}, 0x14000802a00)
        /go/pkg/mod/github.com/onflow/flow-go@v0.31.1-0.20230808172820-f074502a67e3/engine/access/rest/middleware/logging.go:21 +0xbc
net/http.HandlerFunc.ServeHTTP(0x14000802900?, {0x104d72ac0?, 0x1400173a0e0?}, 0xe2c7ec47d300cb58?)
        /usr/local/go/src/net/http/server.go:2109 +0x38
github.com/gorilla/mux.(*Router).ServeHTTP(0x1400093a000, {0x104d72ac0, 0x1400173a0e0}, 0x14000802700)
        /go/pkg/mod/github.com/gorilla/mux@v1.8.0/mux.go:210 +0x19c
github.com/rs/cors.(*Cors).Handler.func1({0x104d72ac0, 0x1400173a0e0}, 0x14000802700)
        /go/pkg/mod/github.com/rs/cors@v1.8.0/cors.go:219 +0x1e4
net/http.HandlerFunc.ServeHTTP(0x0?, {0x104d72ac0?, 0x1400173a0e0?}, 0x1032e0f84?)
        /usr/local/go/src/net/http/server.go:2109 +0x38
net/http.serverHandler.ServeHTTP({0x104d6a1a0?}, {0x104d72ac0, 0x1400173a0e0}, 0x14000802700)
        /usr/local/go/src/net/http/server.go:2947 +0x2c4
net/http.(*conn).serve(0x14000970d20, {0x104d74a30, 0x14000c21ce0})
        /usr/local/go/src/net/http/server.go:1991 +0x560
created by net/http.(*Server).Serve
        /usr/local/go/src/net/http/server.go:3102 +0x444

  --> 4dc519619d3420f382b2c285b1a7b9f65eaae73f483d8359afe2e16ce88d1bdd:17:8
   |
17 |         let rnd = unsafeRandom()
```

File: contracts/HybridCustody.cdc [hunk 2]
```diff
@@ -1023,7 +1023,10 @@ pub contract HybridCustody {
             // NOTE: This path cannot be sufficiently randomly generated, an app calling this function could build a
             // capability to this path before it is made, thus maintaining ownership despite making it look like they
             // gave it away. Until capability controllers, this method should not be fully trusted.
-            let authAcctPath = "HybridCustodyRelinquished".concat(HybridCustody.account.address.toString()).concat(getCurrentBlock().height.toString())
+            let authAcctPath = "HybridCustodyRelinquished"
```
Comment (sisyphusSmiling) [inline contracts/HybridCustody.cdc:1026]:
```suggestion
            let authAcctPath = "HybridCustodyRelinquished_"
```

File: contracts/HybridCustody.cdc (hunk 1 above)
Comment (sisyphusSmiling) [reply contracts/HybridCustody.cdc:1029]: Agreed, random works well here.

FYI I rolled back my CLI version to v1.3.3 and was able to get unsafeRandom() to work. This isn't urgent so if you'd rather wait for the fix to release that'd be fine as well

Comment (sisyphusSmiling) [reply contracts/HybridCustody.cdc:1029]: Agreed, random works well here.

FYI I rolled back my CLI version to v1.3.3 and was able to get unsafeRandom() to work. This isn't urgent so if you'd rather wait for the fix to release that'd be fine as well

Comment (btspoony) [reply contracts/HybridCustody.cdc:1029]: Let's wait for the fix to release

**Comment by btspoony** [comment]
Updated with `unsafeRandom` on new `flow-cli` v1.4.4
@sisyphusSmiling @austinkline

**Review by sisyphusSmiling — APPROVED** [review]
Thank you @btspoony! 🚀

**Comment by sisyphusSmiling** [comment]
Testnet and Mainnet contracts have been updated.

[Testnet updating transaction- c3c31132c1217fb5e993281fa4618e0bd2684a409e1f8ef9ca4d119bddef678d](https://testnet.flowdiver.io/tx/c3c31132c1217fb5e993281fa4618e0bd2684a409e1f8ef9ca4d119bddef678d) 
[Mainnet updating transaction - 7a757e20c55149567715fb7135ef521e02e4345ed19fd4685e2b4b2a4b9f9030](https://www.flowdiver.io/tx/7a757e20c55149567715fb7135ef521e02e4345ed19fd4685e2b4b2a4b9f9030)

**Pull request merged**